zlibWrapper/zstd_zlibwrapper.c

```diff
@@ -500,6 +531,7 @@ ZEXTERN int ZEXPORT z_deflatePending OF((z_streamp strm,
         return deflatePending(strm, pending, bits);
     FINISH_WITH_ERR("deflatePending is not supported!");
```

Comment:
OK I will remove fprintf.

Z_ERRNO = If an error occurred in the file system and not in the compression library, errnum is set to Z_ERRNO and the application may consult errno to get the exact error code.

Z_STREAM_ERROR = if the stream state was inconsistent

Z_BUF_ERROR if no progress is possible (for example avail_in or avail_out was zero) or if there was not enough room in the output buffer when Z_FINISH is used.